crates/flux-desugar/src/lib.rs

```diff
@@ -293,10 +302,10 @@ impl CrateDesugar<'_, '_> {
         for item in specs.flux_items_by_parent.values().flatten() {
             match item {
                 surface::Item::Qualifier(qual) => {
-                    self.desugar_qualifier(qual).collect_err(&mut self.err);
+                    self.desugar_qualifier(&qual).collect_err(&mut self.err);
```

Comment:
This is redundant `qual` already is a reference

```suggestion
                    self.desugar_qualifier(qual).collect_err(&mut self.err);
```